fastlane_core/lib/fastlane_core/test_parser.rb

```diff
@@ -32,7 +41,9 @@ def self.auto_convert(config)
           to_path = path.gsub(".plist", config[:extension])
         end
 
+        # rubocop:disable Lint/MissingRequireStatement
         tp = Trainer::TestParser.new(path)
```

Comment:
This is weird, this file depends on a third party plugin? Help @KrauseFx

Reply:
Uh, that's really weird, I'm pretty sure this doesn't work. Looks like this code is called from SimulatorLauncher?